examples/quickstarts/helidon-quickstart-mp/src/main/java/io/helidon/examples/quickstart/mp/GreetResource.java

```diff
@@ -96,20 +97,27 @@ public JsonObject getMessage(@PathParam("name") String name) {
     /**
      * Set the greeting to use in future messages.
      *
-     * @param newGreeting the new greeting message
+     * @param jsonObject JSON containing the new greeting
      * @return {@link JsonObject}
```

Comment:
This java doc `@return` should change to reflect the new return type.

Reply:
Done